.travis.yml

```diff
@@ -3,17 +3,13 @@
 language: go
 
 go:
- - 1.4
+ - 1.5
```

Comment:
Have to update to `1.5.2`. I thought that this will fetch the latest go 1.5, but apparently it fetches the oldest one.